backend/src/main/java/com/hyooun/wilson/review/domain/Change.java

```diff
@@ -0,0 +1,19 @@
+package com.hyooun.wilson.review.domain;
```

Comment:
ChangeType enum의 MODIFY 오타 수정이 필요

```suggestion
ChangeType enum 수정
```